### PR TITLE
doc: releases: release-notes: 3.6: add CAN release notes for v3.6.0

### PR DESCRIPTION
**Pull request by henrikbrixandersen**
Add CAN release notes for v3.6.0.

### CODE REVIEW

File: doc/releases/release-notes-3.6.rst [hunk 1]
```diff
@@ -456,8 +479,6 @@ Libraries / Subsystems
 
 * LoRa/LoRaWAN
 
-* CAN ISO-TP
-
```
Comment (henrikbrixandersen) [inline doc/releases/release-notes-3.6.rst:460]: @martinjaeger I don't think we've made any changes to ISO-TP for v3.6.0 worth mentioning here?

**Review by fabiobaltieri — APPROVED** [review]

File: doc/releases/release-notes-3.6.rst [hunk 2]
```diff
@@ -215,7 +215,30 @@ Drivers and Sensors
   * Add system call :c:func:`can_get_transceiver()` for getting the CAN transceiver associated with
```
Comment (kartben) [inline doc/releases/release-notes-3.6.rst:215]:
```suggestion
  * Added system call :c:func:`can_get_transceiver()` for getting the CAN transceiver associated with
```

**Review by kartben — APPROVED** [review]
one nit, from a previous commit

**Review by MaureenHelm — APPROVED** [review]

File: doc/releases/release-notes-3.6.rst (hunk 1 above)
Comment (martinjaeger) [reply doc/releases/release-notes-3.6.rst:460]: That's right.

**Review by martinjaeger — APPROVED** [review]

**Review by str4t0m — APPROVED** [review]

**Comment by henrikbrixandersen** [comment]
> one nit, from a previous commit

Thanks. We'll go through all release notes before the release correcting to past tense where needed.

**Pull request merged**